google-cloud-recaptcha_enterprise/Gemfile

```diff
@@ -0,0 +1,8 @@
+source "https://rubygems.org"
+
+gemspec
+
+# WORKAROUND: builds are having problems since the release of 3.0.0
+# pin to the last known good version
+gem "public_suffix", "~> 2.0"
```

Comment:
Is this pin still required?

Reply:
I don't know. It's set by the generator. Running tests for this lib with 3.0.0 didn't fail, but I don't know what the original issue was.

Reply:
I can't get any test failures w/ 3.0.0 on any of the libs with this comment on it.